platform-includes/set-fingerprint/basic/dart.mdx

```diff
@@ -1,14 +1,12 @@
-```dart
-import 'package:sentry/sentry.dart';
-
-FutureOr<SentryEvent?> beforeSend(SentryEvent event, Hint hint) async {
-  if (event.throwable is DatabaseException) {
-    event = event.copyWith(fingerprint: ['database-connection-error']);
-  }
-  return event;
-}
-
-Future<void> main() async {
-  await Sentry.init((options) => options.beforeSend = beforeSend);
-}
+```dart {4-7}
+await SentryFlutter.init(
+  (options) {
+    options.beforeSend = (event, hint) {
+      if (event.throwable is DatabaseException) {
+        event = event.copyWith(fingerprint: ['database-connection-error']);
+      }
+      return event;
+    };
+  },
+);
```

Comment:
```suggestion
```dart {2-6}
await SentryFlutter.init((options) {
  options.beforeSend = (event, hint) {
    if (event.throwable is DatabaseException) {
      event = event.copyWith(fingerprint: ['database-connection-error']);
    }
    return event;
  };
});
```